dist/src/main/distro/bin/mvnd.cmd

```diff
@@ -25,6 +25,11 @@
 @REM   MAVEN_BATCH_PAUSE (Optional) set to 'on' to wait for a key stroke before ending.
 @REM   MAVEN_OPTS        (Optional) Java runtime options used when Maven is executed.
 @REM   MAVEN_SKIP_RC     (Optional) Flag to disable loading of mavenrc files.
+@REM   MVND_CLIENT       (Optional) Control how to select mvnd client to communicate with the daemon:
+@REM                        'auto' (default) - prefer the native client mvnd if it works; otherwise use
+@REM                                           the pure Java client.
+@REM                        'native' - use the native client mvnd.exe

```

Comment:
This is windows, so we can use `mvnd.exe` every where, no?

`if it works` sounds a bit unspecific to me. Could we please somehow concisely describe what the script actually does?

```suggestion
@REM   MVND_CLIENT       (Optional) Control how to select mvnd client to communicate with the daemon:
@REM                        'auto' (default) - prefer the native client mvnd.exe if it suits the current OS and processor architecture; otherwise use
@REM                                           the pure Java client.
@REM                        'native' - use the native client mvnd.exe
```

Reply:
OK~